platform/ui/src/views/StudyList/components/StudyListFilter.js

```diff
@@ -124,87 +124,115 @@ const StudyListFilter = ({
+            <div className="flex flex-row justify-between mb-5 px-12">
+              <div className="flex flex-row">
+                <Typography
+                  variant="h4"
+                  className="text-custom-aquaBright mr-6"
                 >
-                  <FilterLabel
-                    key={name}
-                    label={displayName}
-                    isSortable={
-                      isSortable && numOfStudies <= 100 && numOfStudies > 0
-                    }
-                    isBeingSorted={sortBy === name}
-                    sortDirection={sortDirection}
-                    onLabelClick={() => handleFilterLabelClick(name)}
-                    inputType={inputType}
+                  Study List
+                </Typography>
+                <div className="flex flex-row items-end">
+                  <Button
+                    variant="text"
+                    size="small"
+                    color="inherit"
+                    className="text-custom-blueBright"
+                    startIcon={<Icon name="info-link" className="w-2" />}
                   >
-                    {inputType !== 'none' && (
-                      <Input
-                        className="border-custom-blue mt-2 bg-black"
-                        type="text"
-                        containerClassName="mr-2"
-                        value={currentFiltersValues[name] || ''}
-                        onChange={event => handleFilterValueChange(event, name)}
-                      />
-                    )}
-                  </FilterLabel>
+                    <span className="flex flex-col flex-1">
+                      <span>Learn more</span>
+                      <span className="opacity-50 pt-1 border-b border-custom-blueBright"></span>
+                    </span>
+                  </Button>
                 </div>
-              );
-            }
-          )}
+              </div>
+              <div className="flex flex-row">
+                {isFiltering() && (
+                  <Button
+                    rounded="full"
+                    variant="outlined"
+                    color="primary"
+                    className="text-custom-blueBright border-custom-blueBright mx-8"
+                    startIcon={<Icon name="cancel" />}
+                    onClick={clearFilters}
+                  >
+                    Clear filters
+                  </Button>
+                )}
+                <Typography variant="h4" className="mr-2">
+                  {numOfStudies > 100 ? '>100' : numOfStudies}
+                </Typography>
+                <Typography
+                  variant="h6"
+                  className="text-custom-grayLight self-end pb-1"
+                >
+                  Studies
+                </Typography>
+              </div>
+            </div>
+          </div>
         </div>
       </div>
-    </div>
+      <div className="sticky z-10" style={{ top: 58 }}>
```

Comment:
I am a bit concerned on hardcoding this style to make the sticky working. Once we componentize it and make it reusable it might break.

Reply:
Totally agree! Maybe when we break it into a small component, we can add props like:

```javascript
<Component sticky stickyOffset="58" />
```

What do you think?